### DISCUSSION TITLE
I offer GDPS hosting

### DISCUSSION THREAD

**Discussion by MathieuARS** [Cvolton/GMDprivateServer#840] [General]
Hello! I have a dedicated server and i'm not really using it at its maximum so i offer FREE gdps hosting! I will make everything for you (installing the server & linking the game to the server & mobile version (android only))

It will be much faster than altervista or some others free hosting website

If you want to have your own server just contact me on this discord: https://discord.gg/CkMV5DV

**Comment by DarkGD123** [comment]
i prefer using a free one but thanks for the offer i might do it soon

**Comment by Stazzical** [comment]
dude literally this isnt an issue

**Comment by LBlazen** [comment]
bruh

**Comment by Minosuko** [comment]
may be it a bad

1.You can remote anyonce gdps
2.You can steal password (change at registerGJAccount.php)
3.Security of server will go to 50%

**Reply by ghost** [reply]
I agree. @MathieuARS could just change player information and take over, because he owns the server the GDPS would be hosted on. **I don't think he could steal passwords (passwords are hashed)**

**Reply by Minosuko** [reply]
replace password_hash with base64_encode at "accounts/registerGJAccount.php" then we can steal the password be decode the base64 encoded password

**Comment by fancythedeveloper** [comment]
Literally, just sign up for free on Google Cloud and you don't have to use some other guy's VPS.

**Reply by ghost** [reply]
![image](https://user-images.githubusercontent.com/93995622/226389875-cfa32d62-50a5-413b-81c0-15ab0878baf1.png)

**Comment by ocucor** [comment]
I agree with fancy. I personally use it to host my own GDPS and I don't have any problems with it so far.

**Comment by RXNationGMG** [comment]
R.I.P. GDPS Free Hosting :(

**Reply by Moodygd761** [reply]
fr
